src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy

```diff
@@ -30,13 +90,33 @@ class GatlingPluginExtension implements JvmConfigurable {
             }
         }
 
+        @Input
+        @Optional
         UUID getPackageId() {
             return packageId
         }
 
+        @Input
+        @Optional
         URL getUrl() {
             return url
         }
+
+        @Input
+        @Optional
+        String getClassName() {
+            return className
+        }
+
+        EnterpriseClient getEnterpriseClient(String version) {
```

Comment:
**question:** do you believe we could have only one instance shared?

Reply:
I don't know if it's really useful, as the user would trigger only one task, the instance will be created just once